src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/Interop/VBCompilerOptions.vb

```diff
@@ -131,8 +131,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim.
         ''' <summary>
         ''' Preferred load address.
         ''' </summary>
-        ''' <remarks></remarks>
-        Public dwLoadAddress As UInteger
+        Public dwLoadAddress As UIntPtr
```

Comment:
Our hungarian notation no longer matches the type

Reply:
@jaredpar: this matches the internal interop definition. We'll keep it consistent.

Reply:
Mostly playing around here ;)